HACKING.md

```diff
@@ -0,0 +1,53 @@
+The easiest way to get started hacking on `rpm-ostree` is to
+use the vagrant machine. This is also the set up used for
+our integration tests.
+
+One-time setup
+==============
+
+Starting the vagrant machine is as easy as:
+
+```
+host$ vagrant up
+host$ vagrant ssh
+```
+
+The VM for now uses the official
+[centos/atomic-host](https://atlas.hashicorp.com/centos/boxes/atomic-host)
+box. However, because `rpm-ostree` is tightly coupled with
+other projects such as `ostree` and `libhif`, the code in
+`HEAD` may require a more recent version than available in
+the latest official box.
+
+For this reason, you may want to rebase the VM onto the
+[CentOS Atomic Continuous](https://ci.centos.org/job/atomic-rdgo-centos7/)
+stream, which will contain the latest `HEAD` versions of
+these dependencies (normally within the hour). To rebase,
+simply do:
+
+```
+vm$ sudo rpm-ostree rebase centos-atomic-continuous:centos-atomic-host/7/x86_64/devel/continuous
+vm$ sudo systemctl reboot
+```
+
+If you need to test your code with custom `ostree` or
+`libhif` builds, you have no choice for now other than
+making your own tree (and yum repo for use by the build
```

Comment:
Right, this is fine for testing it manually. I think when I wrote this, I was talking in the context of `vmcheck`, in which case we would prefer to have a clean state to start off from.